.github/workflows/legend-stack-release.yml

```diff
@@ -0,0 +1,86 @@
+# Copyright 2023 Goldman Sachs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Legend Stack Release
+
+on:
+  repository_dispatch:
+    types: [ legend-stack-release ]
+
+env:
+  CI_DEPLOY_USERNAME: ${{ secrets.CI_DEPLOY_USERNAME }}
+  CI_DEPLOY_PASSWORD: ${{ secrets.CI_DEPLOY_PASSWORD }}
+  CI_GPG_PASSPHRASE: ${{ secrets.CI_GPG_PASSPHRASE }}
+  DOCKER_USERNAME: finos
+  DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  MAVEN_OPTS: -Xmx4g
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
```

Comment:
The legend-engine release will not work on this runner.

We should use the 'large' runner used here: https://github.com/finos/legend-engine/blob/master/.github/workflows/release-large-runner.yml#L34

Please adjust the memory parameters as well to be the same as in that action.

Reply:
done